pulp_smash/tests/rpm/api_v2/test_sync_publish.py

```diff
@@ -101,72 +103,98 @@ def test_importer_config(self):
                 self.assertEqual(importers[0][key], body['importer_' + key])
 
 
-class SyncValidFeedTestCase(utils.BaseAPITestCase):
-    """Create an RPM repository with a valid feed and sync it.
+# This class is left public for documentation purposes.
+@unittest2.skip('Base test case.')
+class SyncRepoBaseTestCase(utils.BaseAPITestCase):
```

Comment:
I think I'll leave it as is. It's trivial to move up the directory hierarchy as needed, but at this point, the class is only used in this one module, and moving the class to a `utils` module doesn't help us.